src/java.base/share/classes/java/io/InputStream.java

```diff
@@ -676,7 +683,8 @@ public void close() throws IOException {}
      *
      * <p> Marking a closed stream should not have any effect on the stream.
      *
-     * <p> The {@code mark} method of {@code InputStream} does
+     * @implSpec
+     * The {@code mark} method of {@code InputStream} does
      * nothing.
```

Comment:
Minor nit but the line break can be removed so that "nothing" is on the same line.

Reply:
Sure. (I default to not making such reflow changes in the initial version of a patch to avoid spurious diffs.